src/test/java/nl/tudelft/pixelperfect/event/FireEventTest.java

```diff
@@ -3,6 +3,7 @@
 /**
  * Created by woute on 5/10/2016.
  */
+@SuppressWarnings("PMD")
```

Comment:
We can also leave it as a reminder to remove this SupressWarning after we add tests. Then we have an empty warning sheet to look upon.